nf_core/lint.py

```diff
@@ -1346,11 +1347,11 @@ def _s(some_list):
         table = Table(box=rich.box.ROUNDED)
         table.add_column("[bold green]LINT RESULTS SUMMARY".format(len(self.passed)), no_wrap=True)
```

Comment:
```suggestion
        table.add_column(r"\[bold green]LINT RESULTS SUMMARY".format(len(self.passed)), no_wrap=True)
```

Reply:
Not sure if it should also be escaped here?

Reply:
Sure - this didn't trigger a warning but can't hurt I guess 👍🏻